src/main.cpp

```diff
@@ -1,38 +1,150 @@
-/**
- * Main Program Entry Point
- * Starts and runs the server loop.
- */
 #include "CLI/App.hpp"
-#include "CLI/Formatter.hpp"
 #include "CLI/Config.hpp"
+#include "CLI/Formatter.hpp"
+#include "services/service_manager.h"
 #include "src/version.h"
+#include <algorithm>
+#include <fmt/format.h>
+
+void printServerList(const std::vector<record_server_t>& servers) {
```

Comment:
This function is kind of a mess. This is just an attempt to align the list nicely